src/bp-moderation/screens/single/admin/report-single.php

```diff
@@ -156,7 +156,8 @@ class="<?php echo ( ! empty( $error ) ) ? 'error' : 'updated'; ?>">
 											</td>
 											<td>
 												<?php
-												printf( '<strong>%s</strong>', wp_kses_post( bp_core_get_userlink( $moderation_request_data->item_id ) ) );
+												$bypass_avatar_url = add_query_arg( array( 'modbypass' => 1 ), bp_core_get_user_domain( $moderation_request_data->item_id ) );
```

Comment:
@janakkaneriya007 This is wrong you should use BP_Moderation_Members ->get_permalink function to user link with modbypass argument